src/components/draggable/draggable-item.style.ts

```diff
@@ -1,14 +1,20 @@
 import styled from "styled-components";
-import { padding, margin } from "styled-system";
+import { padding, margin, PaddingProps, MarginProps } from "styled-system";
 
 import { baseTheme } from "../../style/themes";
 import Icon from "../icon";
 
-const StyledDraggableContainer = styled.div`
+export type StyledDraggableContainerProps = MarginProps;
+
+const StyledDraggableContainer = styled.div<StyledDraggableContainerProps>`
   ${margin}
 `;
 
-const StyledDraggableItem = styled.div`
+export interface StyledDraggableItemProps extends PaddingProps {
```

Comment:
This is fine but if we go with my suggestion about `DraggableItemProps` extending `PaddingProps` then this doesn't need to be exported anymore